charts/gardener/controlplane/values.yaml

```diff
@@ -284,6 +284,13 @@ global:
         # issuer: https://issuer.gardener.cloud
         minExpiration: 1h
         maxExpiration: 48h
+        signingKey:
+        # tlsSecretName: workload-identity-secret # must contain 'tls.key' key
+          tls:
+            key: |
+              -----BEGIN RSA PRIVATE KEY-----
+              ...
+              -----END RSA PRIVATE KEY-----
```

Comment:
Why do we need a "tls" configuration? Shouldn't this just be a

```
signing:
  secretName: <Name of the secret>
  key: <OR RAW KEY>
```

Reply:
Good point, I was trying to align it with https://github.com/gardener/gardener/blob/1745a0984f84e9ea5818dbfe49fa044d09c30082/charts/gardener/controlplane/values.yaml#L88-L97, but here we do not need full TLS spec. I will apply the suggest change.